webpack.config.js

```diff
@@ -219,23 +225,37 @@ const webpackConfig = {
 			},
 		} ),
 		new WebpackRTLPlugin( {
-			filename: './dist/[name]/style-rtl.css',
+			filename: './[name]/style-rtl.css',
 			minify: {
 				safe: true,
 			},
 		} ),
 		new MiniCssExtractPlugin( {
-			filename: './dist/[name]/style.css',
+			filename: './[name]/style.css',
 		} ),
 		new CopyWebpackPlugin(
 			wcAdminPackages.map( ( packageName ) => ( {
 				from: `./packages/${ packageName }/build-style/*.css`,
-				to: `./dist/${ packageName }/`,
+				to: `./${ packageName }/`,
 				flatten: true,
 				transform: ( content ) => content,
 			} ) )
 		),
-	],
+		new MomentTimezoneDataPlugin( {
+			startYear: 2000, // This strips out timezone data before the year 2000 to make a smaller file.
```

Comment:
I'll let @superdav42 answer - it's the same year chosen in his WCS PR: https://github.com/Automattic/woocommerce-services/pull/1994/commits/3738a398fc197236c300938234bba9a6db0ce3d2#diff-11e9f7f953edc64ba14b0cc350ae7b9dR222

My guess is that it's an arbitrary margin of safety before WC was created.

Reply:
I think I picked it because it was in the example and it's a few years before WordPress was released. It's admittedly very conservative but since I'm not familiar with how moment is used in wc-admin I picked a safe value.